assets/dev/js/editor/elements/manager.js

```diff
@@ -0,0 +1,80 @@
+import ElementBase from './types/base/element-base';
+
+import * as elementsTypes from './types/';
+
+export default class ElementsManager {
+	/**
+	 * Registered elements types.
+	 *
+	 * @type {Object.<ElementBase>}
+	 */
+	elementTypes = {};
+
+	constructor() {
+		this.registerElements();
+	}
+
+	/**
+	 * Function getElementType().
+	 *
+	 * Get element type by elType, and widgetType, if exact widgetType is not found
+	 * the method will return the element type ( WidgetBase ).
+	 *
+	 * @param {string} elType
+	 * @param {string|false} widgetType
+	 */
+	getElementType( elType, widgetType = false ) {
+		let key = 'element-' + elType,
+			result = this.elementTypes[ key ];
+
+		if ( 'widget' === elType ) {
+			key = 'widget-' + widgetType;
+
+			if ( this.elementTypes[ key ] ) {
+				result = this.elementTypes[ key ];
+			}
+		}
+
```

Comment:
Please use `widgetType || elType`, in order to be prepared to unify the widget with the elements.

Reply:
Handled https://github.com/elementor/elementor/pull/18297